src/exceedances/README.md

```diff
@@ -30,6 +30,7 @@
 
 * Run script
     * `python main.py`
+    * or by explicitly stating the tenant `python main --tenant=kcca`
```

Comment:
I think we forgot to add the file extension on this entry file....should it be: python main.py --tenant=kcca

Reply:
fixed

Reply:
Great, thanks!